README.md

```diff
@@ -1 +1,92 @@
-# bahmni-metabase
\ No newline at end of file
+# bahmni-metabase
+
+## Table of Contents
+* [Metabase Overview](#metabase-overview)
+* [Admin user creation for Metabase](#admin-user-creation-for-metabase)
+* [Connecting Metabase with Database](#connecting-metabase-with-database)
+* [Configure Predefined Reports in Metabase](#configure-predefined-reports-in-metabase)
+* [Starting Metabase from Bahmni Docker](#starting-metabase-from-bahmni-docker)
+
+
+# Metabase Overview
+
+Metabase is an open-source business intelligence tool. Metabase lets you ask questions about your data, and displays answers in formats that make sense, whether that’s a bar chart or a detailed table.
+
+You can save your questions, and group questions into handsome dashboards. Metabase also makes it easy to share questions and dashboards with the rest of your team.
+
+As a part of our initiative we have used [v0.44.6](https://hub.docker.com/layers/metabase/metabase/v0.44.6/images/sha256-527a30f88f79a90bba0951d352a9fcbdecbf5f1b011a41b30e7a074265549168?context=explore) version of Metabase image.
+
+# Admin user creation for Metabase
+The first user of metabase will be created as an Admin user and is pre-configured for [metabase profile](https://github.com/Bahmni/bahmni-docker/blob/master/bahmni-standard/docker-compose.yml) in the docker-compose.yml of bahmni-docker by adding the following configurational data for admin.
```

Comment:
Also, docker-compose and metabase documentation is supposed to be put here: 
https://bahmni.atlassian.net/wiki/spaces/BAH/pages/3117482143/Metabase+Configuration+docker

This is the wiki page for all docker-compose documentation for how to "consume" metabase documentation. In README just provide what is needed for creating a docker image, etc.

Reply:
We will move the docker related section to wiki.